### ISSUE TITLE
ThemePackage.getStylesheetsPath is deprecated.

### ISSUE THREAD

**Issue by dhilipsiva** [jesseweed/seti-syntax#97]
Store package style sheets in the `styles/` directory instead of `stylesheets/` in the `seti-syntax` package
```
ThemePackage.getStylesheetsPath (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/package.js:430:9)
ThemePackage.getStylesheetPaths (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/src/package.js:441:32)
```

**Comment by asantos3** [comment]
Fixed in version 0.4.0, update and close this.

**Comment by dhilipsiva** [comment]
Oh foosh! Didn't knew that. Thanks :)
